pkg/jobrunaggregator/jobrunbigqueryloader/uploader.go

```diff
@@ -248,7 +280,6 @@ func (o *jobRunLoaderOptions) Run(ctx context.Context) error {
 	}
 
 	if err := o.uploadJobRun(ctx, jobRun); err != nil {
-		o.logger.WithError(err).Error("failed to upload jobrun to bigquery")
```

Comment:
why suppress the error?

Reply:
This was from a recent change I did and got carried away, the error was logged at three levels. I don't have a personal rule on consistency for logging them low level or higher up so nothing gets missed, but in this case opted to kill off this higher level logging as it was covered lower.